plugins/joblib_launcher/README.md

```diff
@@ -0,0 +1,33 @@
+# Joblib Launcher
+
+This plugin provides a launcher for [embrassingly parallel](https://en.wikipedia.org/wiki/Embarrassingly_parallel) tasks. It is based on [`Joblib.Parallel`](https://joblib.readthedocs.io/en/latest/parallel.html).
+
+An example app using the plugin is provided in `plugins/joblib_launcher/example`. Starting the app with `python my_app.py --multirun task=1,2,3,4,5` will launch five parallel executions.
+
+The maximum number of concurrently running jobs can be restricted by providing `hydra.launcher.params.n_jobs`. In the example application, `n_jobs` is set to -1, which means that all available CPUs may be used:
+
+`hydra_plugins/joblib_launcher/conf/hydra/launcher/example.yaml`:
+```yaml
+hydra:
+  launcher:
+    class: hydra_plugins.joblib_launcher.JoblibLauncher
+    params:
+      n_jobs: -1
+```
+
```

Comment:
You can just provide a link to this file in the example. normally users will not need to copy paste the whole thing.
document individual params like n_jobs and provide a general example for how to override (in command line as you did, or in the config file - and in that case there i no need to repeat the class name, only the overridden parameters.